src/WallToolPaths.cpp

```diff
@@ -51,13 +51,14 @@ WallToolPaths::WallToolPaths(const Polygons& outline, const coord_t bead_width_0
 const std::vector<VariableWidthLines>& WallToolPaths::generate()
 {
     const coord_t allowed_distance = settings.get<coord_t>("meshfix_maximum_deviation");
+    const coord_t open_close_distance = settings.get<coord_t>("min_wall_line_width") / 4;
```

Comment:
Nitpicking; This deserves documentation. Why is the open_close_distance the value that it has? We really wont remember why this was done in the future otherwise :P